app/src/main/java/org/squidfish/tuplenback/di/RepositoryModule.kt

```diff
@@ -1,6 +1,13 @@
 package org.squidfish.tuplenback.di
 
 import org.koin.core.annotation.Module
+import org.koin.core.annotation.Single
+import org.squidfish.tuplenback.data.LocalStorageRepository
+import org.squidfish.tuplenback.data.room.AppDatabase
 
 @Module
-class RepositoryModule
+class RepositoryModule {
+
+    @Single
+    fun createRepository(db: AppDatabase): LocalStorageRepository = LocalStorageRepository(db)
```

Comment:
You provide the repository manually, so there indeed is no reason to have the class declaration annotated with `@Single`